### DISCUSSION TITLE
Window and Plugins

### DISCUSSION THREAD

**Discussion by Tonaion02** [bevyengine/bevy#14522] [Q&A] [answered]
What are the necessary plugin to make a window in Bevy without use DefaultPlugins?

**Comment by atornity** [comment, marked as answer]
it seems to work with just `WindowPlugin`, `WinitPlugin` and `AccessibilityPlugin`.

**Reply by alice-i-cecile** [reply]
You may want the rest of `MinimalPlugins` in there to make sure things like time work correctly too.

**Reply by Tonaion02** [reply]
I don't know why but with this code:
![image](https://github.com/user-attachments/assets/0aa4d1d6-a347-47fb-ab8d-7478a577ad86)
the program panicks.
Panick message:
![image](https://github.com/user-attachments/assets/525eccc5-dba5-444b-87be-d403030208ea)

**Reply by alice-i-cecile** [reply]
I suspect your plugin ordering is wrong :( See #1255 for a discussion of this problem. Looks like `AccessiblityPlugin` needs to occur before `WindowPlugin`, reading the error output.
